### ISSUE TITLE
getOrgSubmissionsList and getDeliveries times out in Prod.

### ISSUE THREAD

**Issue by jimduff-usds** [CDCgov/prime-reportstream#7658]
**Describe the bug**
getOrgSubmissionsList times out in Prod.

```
Timeout value of 00:30:00 was exceeded by function: Functions.getOrgSubmissionsList
Problem Id:Microsoft.Azure.WebJobs.Host.FunctionTimeoutException at Microsoft.Azure.WebJobs.Host.Executors.FunctionExecutor+<TryHandleTimeoutAsync>d__35.MoveNext
```

**Impact**
Not only slow response but this is triggering exceptions.

**To Reproduce**

Get submissions on the UI for a sender that has sent a ton of data.

**Expected behavior**
Fast response. (< 10 seconds)

**Comment by jimduff-usds** [comment]
we were not able to recreate this slowness in either local or in Staging.

We are seeing lots of exceptions in Prod.

Next step is to factor out just the SQL, then pair on running this in Prod.

**Comment by jimduff-usds** [comment]
Here's a different but similar slowness:

```
Uncaught not-found: Request failed with status code 404
Timeout value of 00:30:00 was exceeded by function: Functions.getDeliveries
Problem Id:Microsoft.Azure.WebJobs.Host.FunctionTimeoutException at Microsoft.Azure.WebJobs.Host.Executors.FunctionExecutor+<TryHandleTimeoutAsync>d__35.MoveNext
```

**Comment by thetaurean** [comment]
The problem for both of these queries was a faulty brin index on the `report_file` table. The fix is being tracked in #7920.
